conf/supervisord.conf.jnj

```diff
@@ -257,4 +349,6 @@ command=python -m workers.repomirrorworker.repomirrorworker
 autostart = {{ config['repomirrorworker']['autostart'] }}
 stdout_events_enabled = true
 stderr_events_enabled = true
-# EOF NO NEWLINE
```

Comment:
Please keep this; no newline at eof in this file.

Reply:
Is that a comment or something used by supervisor?